kusto/ingest/internal/resources/resources.go

```diff
@@ -126,9 +126,11 @@ type token struct {
 
 // Manager manages Kusto resources.
 type Manager struct {
-	client    mgmter
-	done      chan struct{}
-	resources atomic.Value // Stores Ingestion
+	client               mgmter
+	done                 chan struct{}
+	resources            atomic.Value // Stores Ingestion
+	kustoToken           token
+	kustoTokenExpiration time.Time
```

Comment:
I know this PR is already merged, but small comment -
"kustoTokenExpiration" sounds like it's when the token expires. I suggest "kustoTokenCacheExpiration".